java/driver/jdbc/src/main/java/org/apache/arrow/adbc/driver/jdbc/ObjectMetadataBuilder.java

```diff
@@ -21,11 +21,10 @@
 import java.sql.DatabaseMetaData;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```

Comment:
nit: I'd prefer to avoid star imports if possible

Reply:
agreed, not sure how I missed this